### ISSUE TITLE
DELETE /api/v3/auth_username 如何开启 Basic Auth

### ISSUE THREAD

**Issue by Carl-DS** [emqx/emqx#2570]
#### Environment

- OS: Centos7
- Erlang/OTP: 
- EMQ: V3.1.1

#### Description

*A description of the issue*
在没有Basic Auth 下可以使用RESTful API /api/v3/auth_username,
但我希望 DELETE /api/v3/auth_username 使用Basic Auth该如何做?
浏览了一遍文档并未发现相关内容

**Comment by HJianBo** [comment]
The Basic Authentication must be required by RESTful API.

So What's meaning about `在没有Basic Auth 下可以使用RESTful API /api/v3/auth_username`?

**Comment by Carl-DS** [comment]
```
{
        "name": "delete_username",
        "method": "DELETE",
        "path": "/auth_username/:username",
        "descr": "Delete username in the cluster"
}
```
By default, this API can be accessed without Basic Authentication.

**Comment by Leen15** [comment]
Same problem here, anybody with the dashboard url can create or delete users without authentication.

**Comment by HJianBo** [comment]
Hi, guys. Sorry for late reply.

I have checked it. This problem is reproduced at '18083' port of Dashboard API, not 8080 of REST API.

I have fixed it, and it will be released at v3.2.0
